src/renderer/marketplace/bank-accounts/checkout/payment-complete.jsx

```diff
@@ -94,11 +95,25 @@ export const BankAccountsPaymentComplete = withStyles(styles)(props => {
 							<Typography variant="body1" gutterBottom>
 								Thank you for payment!
 							</Typography>
-							<Typography variant="body2" gutterBottom>
-								Please click the continue button and select your preferred Bank to
-								continue the process. If you have any questions in the meantime, you
-								can reach us at:
-							</Typography>
+							{simpleFlow && (
+								<Typography variant="body2" gutterBottom>
+									One of our managers is reviewing the information you submitted
+									and{' '}
+									<strong>
+										will contact you shortly on the e-mail you provided{' '}
+									</strong>
+									, to continue the process. If you have any questions in the
+									meantime, you can reach us at:
+								</Typography>
+							)}
+
+							{!simpleFlow && (
```

Comment:
Works for now. 
But I this is a very good example of where your idea of composable UI flows would be used.